mysql-test/include/temp_table_size_counter_perl.inc

```diff
@@ -0,0 +1,22 @@
+--perl
+my $dir = $ENV{'MYSQLD_TMPDIR'};
+my $size;
+my $found = 0;
+
+opendir(DIR, $dir) or die $!;
+while (my $file = readdir(DIR))
+{
+  next unless ($file =~ m/\.MYD$/);
```

Comment:
It doesn't. But also, this will never work as it did for myisam: we have a buffer pool with a larger initial size, and then it increases in larger chunks. That's why I don't call it from the ib version of the test.